app/containers/markdown/MessageBody/index.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Quote from './Quote';
+import Paragraph from './Paragraph';
+import Heading from './Heading';
+import Code from './Code';
+import Link from './Link';
+import BigEmoji from './BigEmoji';
+import OrderedList from './OrderedList';
+import UnorderedList from './UnorderedList';
+
+const isBigEmoji = tokens => tokens.length === 1 && tokens[0].type === 'BIG_EMOJI';
+
+const Body = ({
+	tokens, mentions, channels, navToRoomInfo, style
+}) => {
+	if (isBigEmoji(tokens)) {
+		return <BigEmoji value={tokens[0].value} />;
+	}
+
+	return (
```

Comment:
You're not applying styles to text.
New markdown should be rendered with the same styles of the old one.
The only difference is the parser.